packages/cozy-client/src/helpers/settings.js

```diff
@@ -0,0 +1,188 @@
+import CozyClient from '../CozyClient'
+import fetchPolicies from '../policies'
+import { Q } from '../queries/dsl'
+
+const defaultFetchPolicy = fetchPolicies.olderThan(10 * 1000)
+
+/**
+ * Query the cozy-app settings corresponding to the given slug and
+ * extract the value corresponding to the given `key`
+ *
+ * @param {CozyClient} client - Cozy client instance
+ * @param {string} slug - the cozy-app's slug containing the setting (can be 'instance' for global settings)
+ * @param {string} key - The name of the setting to retrieve
+ * @param {any} [defaultValue] - The default value of the setting if it does not exist
```

Comment:
I'm not sure about the value of having this `defaultValue`. What does it bring to pass it as an argument? The app could just deal with it itself, no? 
I fell like it brings more confusion/API complexity than value

Reply:
This is because I already felt on that trap here: https://github.com/cozy/cozy-home/pull/2117#discussion_r1560734796

I suppose that with an API that simplify the settings access it is easier to forget to check those kind of details.

But I'm opened to remove it if you still think this is not worth

Reply:
In my opinion, it's better to keep the API simple and avoid having too many arguments, especially for this simple thing that can be easily handled afterwards.
But this is a matter of opinion, feel free to keep it if you feel it